src/hott/02-homotopies.rzk.md

```diff
@@ -12,28 +12,36 @@ This is a literate `rzk` file:
 #section homotopies
 
 #variables A B : U
+```
 
--- The type of homotopies between parallel functions.
+```rzk title="The type of homotopies between parallel functions"
 #def homotopy
   (f g : A → B)
   : U
   := (a : A) → (f a = g a)
+```
 
--- The reversal of a homotopy
+```rzk title="The reversal of a homotopy"
 #def homotopy-rev
   (f g : A → B)
   (H : homotopy f g)
   : homotopy g f
   := \ a → rev B (f a) (g a) (H a)
+```
 
--- Homotopy composition is defined in diagrammatic order like concat but unlike composition.
+```rzk
 #def homotopy-composition
   (f g h : A → B)
   (H : homotopy f g)
   (K : homotopy g h)
   : homotopy f h
   := \ a → concat B (f a) (g a) (h a) (H a) (K a)
+```
+
+Homotopy composition is defined in diagrammatic order like `concat` but unlike
```

Comment:
Ah, I agree with that suggestion. I'm renaming things in #41, so I can make the change in that PR.

Reply:
Personally, I like `2-path`, but I do not have a strong preference.

I also prefer full `homotopy` to `htpy` and I would replace all `htpy` with `homotopy` (or `2-path` where appropriate).

Reply:
Currently, I've resolved the conflict by using `eq` mostly, and `homotopy-composition` is now called `concat-homotopy` in #41. Let me know if you want any of the new suggestions changed.

Reply:
We could also go for `path-htpy/homotopy`, `fun-htpy`, and `arr-htpy` as a general naming scheme. I like this because it's uniform while still unambiguous.